physical_infra_provider_configuration/automation/topics/creating_configuring_assigning_policy.adoc

```diff
@@ -0,0 +1 @@
+=== Creating, Configuring &nbsp;and Assigning a new Policy
```

Comment:
Did you mean to leave this section empty?

Reply:
Also, if you keep this section, you may want to remove _&nbsp_ .

Reply:
done